src/components/JournalCard/EmojiData.js

```diff
@@ -0,0 +1,35 @@
+import happy from './images/happy.svg';
+import unhappy from './images/unhappy.svg';
+import inLove from './images/in-love.svg';
+import angry from './images/angry.svg';
+import surprised from './images/surprised.svg';
+
+const Feelings = [
```

Comment:
Maybe if you used a different data type it would be easier, like an object with all the emotions and each emotion has its own object as well.
```js
{
  happy: {
    id: 1,
    photo: happy,
    alt: 'happy emoji'
  }
  sad: ...... // and so on
}
```
**Note**: this is just a suggestion by the way :smiley: